packages/devtools_app/lib/src/debugger/flutter/debugger_screen.dart

```diff
@@ -64,8 +64,8 @@ class DebuggerScreenBodyState extends State<DebuggerScreenBody>
   Stack stack;
 
   @override
-  void initState() {
-    super.initState();
+  void didChangeDependencies() {
```

Comment:
The serviceManager.service.getScripts code should move into initState or better yet that logic should move to the controller.
Right now there are a fair number of bugs in that logic related to lifecycles if new isolates load or dependencies change.